src/firebase.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom'
+import * as firebase from 'firebase';
+
+const firebaseConfig = {
+    apiKey: "AIzaSyDqAp9356bLfSL-IhTX2NMTcerekmCTYgk",
+    authDomain: "bermuda-6e374.firebaseapp.com",
+    databaseURL: "https://bermuda-6e374.firebaseio.com",
+    projectId: "bermuda-6e374",
+    storageBucket: "bermuda-6e374.appspot.com",
+    messagingSenderId: "1093935326369",
+    appId: "1:1093935326369:web:d1660d17eb45f22a8efce0",
+    measurementId: "G-LFVJCPWDCY"
+};
```

Comment:
Should we have the apiKey &c exposed in the code like this?

Reply:
https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public Neat!